notebooks/ivf_flat_example.ipynb

```diff
@@ -0,0 +1,803 @@
+{
```

Comment:
<p>Can we put a quick link to the corresponding portion of the pylibraft docs here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>added</p>